auditlog_tests/tests.py

```diff
@@ -1797,3 +1798,29 @@ def test_serialize_related_with_kwargs(self):
                 "value": 11,
             },
         )
+
+
+class TestAccessLog(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username="test_user", is_active=True)
+        self.obj = SimpleModel.objects.create(text="For admin logentry test")
+
+    def test_access_log(self):
+        self.client.force_login(self.user)
+        content_type = ContentType.objects.get_for_model(self.obj.__class__)
+
+        # Check for log entries
+        qs = LogEntry.objects.filter(content_type=content_type, object_pk=self.obj.pk)
+        old_count = qs.count()
+
+        self.client.get(reverse("simplemodel-detail", args=[self.obj.pk]))
+        new_count = qs.count()
+        self.assertEqual(new_count, old_count + 1)
+
+        log_entry = qs.latest()
+        self.assertEqual(int(log_entry.object_pk), self.obj.pk)
+        self.assertEqual(log_entry.actor, self.user)
+        self.assertEqual(log_entry.content_type, content_type)
+        self.assertEqual(
+            log_entry.action, LogEntry.Action.ACCESS, msg="Action is 'ACCESS'"
+        )
```

Comment:
Please test the content of the `changes` field in this test.

Reply:
@simonkern have you checked this comment?

Reply:
I've just received an email notification about this change request, but cannot find your comment quoted in the mail, so I'll reply here:

Yes I checked that comment and introduced a check:
https://github.com/s2k-digital/django-auditlog/blob/master/auditlog_tests/tests.py#L1827